docs/topics/forms/formsets.txt

```diff
@@ -511,6 +511,20 @@ On the other hand, if you are using a plain ``FormSet``, it's up to you to
 handle ``formset.deleted_forms``, perhaps in your formset's ``save()`` method,
 as there's no general notion of what it means to delete a form.
 
+``can_delete_extra``
+~~~~~~~~~~~~~~~~~~~~
+
+.. attribute:: BaseFormSet.can_delete_extra
+
+Default: ``True``
+
+While setting ``can_delete=True``, specifying ``can_delete_extra=False`` will
+remove the option to delete extra forms.
+
+.. versionadded:: 1.10
```

Comment:
Since this applies to the attribute, you can put `.. versionadded:: 1.10` below `.. attribute::` without the "... was added" sentence.